mambaSharedFramework/Pantos-Generic Playlist Parsing/Pantos-Generic Tag Validators/GenericDictionaryTagValidator.swift

```diff
@@ -19,39 +19,39 @@
 
 import Foundation
 
-/// Generically validates dictionary style HLS tags.
-public class GenericDictionaryTagValidator: HLSTagValidator {
+/// Generically validates dictionary style playlist tags.
+public class GenericDictionaryTagValidator: PlaylistTagValidator {
     
     private let dictionaryValueIdentifiers: [DictionaryTagValueIdentifier]
-    private let tag: HLSTagDescriptor
+    private let tag: PlaylistTagDescriptor
     
     /**
      Initializer for GenericDictionaryTagValidator
      
-     - parameter tag: The HLSTagDescriptor for this tag.
+     - parameter tag: The TagDescriptor for this tag.
```

Comment:
`TagDescriptor` is used in this swiftdoc, `PlaylistTagDescriptor` is used in the function definition below.